beamer/tests/util.py

```diff
@@ -162,29 +178,34 @@ def make_request(
     amount,
     validity_period=3600,
     target_chain_id=None,
-    zero_fees=True,
+    fee_data=(0, 0, 0),
```

Comment:
This is mostly used with `"default"`. Can you make this the default value instead, so we can omit this in most cases?

Reply:
Unfortunately, it rather the other way around, vast majority of our tests assume zero fees, which is why I left it as is. :cry:

Reply:
> but why do we need this special case then?

`(0, 0, 0)` is the previous "default" i.e. to make the existing tests continue to work as before.
Originally we said we'd use zero fees to make calculation easier.

But if we want to use the original fee data from the contract, we (unfortunately) have to explicitly request that.
Previously, that was done by saying `zero_fees=False`, now that is done by saying `fee_data="default"`.

Reply:
It occurred to me that maybe `fee_data="contract"` would perhaps convey the meaning better -- as in, "use whatever the fee data is defined by the contract". Would that seem better to you?

Reply:
In direct comparison I think `"standard"` is better. Sorry for all the noise here.

Reply:
Changed to use `"standard"`.